src/adapters/appnexus.js

```diff
@@ -201,6 +202,14 @@ AppNexusAdapter = function AppNexusAdapter() {
         bidmanager.addBidResponse(placementCode, bid);
       }
 
+      if (!usersync) {
+        var iframe = utils.createInvisibleIframe();
+        iframe.src = '//acdn.adnxs.com/ib/static/usersync/v3/async_usersync.html';
+        document.body.appendChild(iframe);
```

Comment:
there's probably a small chance that `document.body` is undefined, so wrap in try/catch.